src/components/table/JadeTable.jsx

```diff
@@ -76,7 +76,7 @@ export class JadeTable extends React.PureComponent {
     handleFilter: PropTypes.func,
     rows: PropTypes.arrayOf(PropTypes.object),
     summary: PropTypes.bool,
-    totalCount: PropTypes.number.isRequired,
+    totalCount: PropTypes.number, // TODO why getting undefined sometimes?
```

Comment:
Are you pulling this from the response? It could have been from branches that didn't have the new counting code. It should never be undefined in the response. When you initialize this is there a default value you're passing in from the store?